### DISCUSSION TITLE
Main Menu Location

### DISCUSSION THREAD

**Discussion by DanyaoWang-001167576** [ian-tiggeman/T24S2T1EXTG08#4] [General]
Hi Ian,
After cloning the GitHub repository to my local repo and opening the Calculator.sln file, the page that appears is a calculator page. I couldn't locate the 'main menu' page. I have checked all the folders and pages in the solution but couldn't find it. Could you help me identify where the 'main menu' page is located?

Regards, 
Cara
![Screenshot 2024-09-06 131807](https://github.com/user-attachments/assets/3a5276a2-c037-4dec-bd09-3749bddf4342)

**Comment by ian-tiggeman** [comment]
Hi Cara,

I actually renamed 'MainPage'.xaml to 'MathCalculator.xaml' (or similar) and added a MainMenu.xaml page in the same folder, so it appears you are looking at an older version of the repo?
Please close all files and make sure you git-checkout the development branch before opening the files.
It should be there.
Let me know how you go.

Regards,
Ian

________________________________
From: Danyao Wang ***@***.***>
Sent: 06 September 2024 13:33
To: ian-tiggeman/T24S2T1EXTG08 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [ian-tiggeman/T24S2T1EXTG08] Main Menu Location (Discussion #4)


Hi Ian,
After cloning the GitHub repository to my local repo and opening the Calculator.sln file, the page that appears is a calculator page. I couldn't locate the 'main menu' page. I have checked all the folders and pages in the solution but couldn't find it. Could you help me identify where the 'main menu' page is located?

Regards,
Cara
Screenshot.2024-09-06.131807.png (view on web)<https://github.com/user-attachments/assets/3a5276a2-c037-4dec-bd09-3749bddf4342>
